game/scripts/npc/heroes/alchemist.txt

```diff
@@ -0,0 +1,12 @@
+"DOTAHeroes"
+{
+  //=================================================================================================================
+  // HERO: Alchemist
+  //=================================================================================================================
+  "npc_dota_hero_alchemist"
+  {
+    // Abilities
+    //-------------------------------------------------------------------------------------------------------------
+    "AttributeStrengthGain"		"3.000000"
```

Comment:
Mjollnir and Armlet were buffed recently (items that Alchemist likes) so there is no need for this.